resources/shared/test-invoker.mjs

```diff
@@ -48,8 +67,58 @@ export class RAFTestInvoker extends TestInvoker {
     }
 }
 
+class AsyncRAFTestInvoker extends BaseRAFTestInvoker {
+    static mc = new MessageChannel();
```

Comment:
Why does this need to be a static variable?

Reply:
Oh I see. Maybe but can't we use an instance variable in that case? We don't re-create TestInvoker for each suite / step, right?

Reply:
each test creates a new instance:

https://github.com/WebKit/Speedometer/blob/main/resources/shared/test-runner.mjs#L74-L75